FileAPI/Blob-methods-from-detached-frame.html

```diff
@@ -48,6 +49,14 @@
     assert_equals(charCodeBufferToString(charCodeBuffer), "bar");
 }, "arrayBuffer()");
 
+promise_test(async () => {
+    const { bytes } = await BlobPrototypeFromDetachedFramePromise;
+    const blob = new Blob(["bar"]);
+
+    const charCodeBytes = await bytes.call(blob);
+    assert_equals(charCodeArrayToString(charCodeBuffer), "bar");
```

Comment:
With this change the WebKit implementation passes these tests. Thanks for writing them!
```suggestion
    assert_equals(charCodeArrayToString(charCodeBytes), "bar");
```